### PR TITLE
zh-translation: about/community/customers/index.md

### PR DESCRIPTION
**Pull request by MXuDong**
[ ] Configuration Infrastructure
[X] Docs
[ ] Installation
[ ] Networking
[ ] Performance and Scalability
[ ] Policies and Telemetry
[ ] Security
[ ] Test and Release
[ ] User Experience
[ ] Developer Infrastructure

### CODE REVIEW

**Comment by MXuDong** [comment]
@googlebot I signed it!

File: content/zh/about/community/customers/index.md [hunk 1]
```diff
@@ -1,40 +1,78 @@
...
+title: 谁正在使用Istio
+description: 一些使用Istio的公司。
 weight: 30
 icon: customers
 keywords: [community]
 skip_seealso: true
 ---
-
-以下是 Istio 的用户。也许你也应该试试？
+以下列出我们知道的正在使用Istio的组织，如果你想编辑这个页面, [请提交一个PR!](https://github.com/istio/istio.io/tree/master/content/en/about/community/customers)
```
Comment (kebe7jun) [inline content/zh/about/community/customers/index.md:9]: 看看 https://istio.io/latest/zh/about/contribute/style-guide/ 吧。

File: content/zh/about/community/customers/index.md [hunk 2]
```diff
@@ -1,40 +1,78 @@
 ---
-title: Istio in Action
-description: 使用 Istio 的用户。
+title: 谁正在使用Istio
+description: 一些使用Istio的公司。
```
Comment (kebe7jun) [inline content/zh/about/community/customers/index.md:3]: 就叫用户吧。

File: content/zh/about/community/customers/index.md (hunk 1 above)
Comment (MXuDong) [reply content/zh/about/community/customers/index.md]: 已经修改

File: content/zh/about/community/customers/index.md (hunk 2 above)
Comment (MXuDong) [reply content/zh/about/community/customers/index.md]: 已经修改

File: content/zh/about/community/customers/index.md [hunk 3]
```diff
@@ -1,40 +1,79 @@
...
+title: 谁正在使用 Istio
 description: 使用 Istio 的用户。
 weight: 30
 icon: customers
 keywords: [community]
 skip_seealso: true
 ---
-
-以下是 Istio 的用户。也许你也应该试试？
+以下向您展示正在使用 Istio 的组织，如果您想编辑这个页面, [请提交PR!](https://github.com/istio/istio.io/tree/master/content/en/about/community/customers)
```
Comment (kebe7jun) [inline content/zh/about/community/customers/index.md:9]:
```suggestion
以下向您展示正在使用 Istio 的用户，如果您想编辑这个页面, [请提交 PR！](https://github.com/istio/istio.io/tree/master/content/en/about/community/customers)
```

**Review by kebe7jun — APPROVED** [review]

**Review by kebe7jun — APPROVED** [review]

**Pull request merged**